test/profileCache.js

```diff
@@ -207,6 +210,41 @@ describe('profile cache', function() {
     });
   });
 
+  it('should invalidate cache when auth-server profileChangedAt is greater than cached version', function (done) {
+    const userid = uid();
+    this.timeout(5000);
+    Server.server.initialize(() => {
+      let lastModified;
+      // first req, store last modified header
+      mock.profileChangedAt(MOZILLA_EMAIL, PROFILE_CHANGED_AT);
+      makeProfileReq(userid)
+      .then(res => {
+        assert.ok(res.headers['last-modified']);
+        lastModified = res.headers['last-modified'];
+        return P.delay(500);
+      })
+      .then(() => {
+        // second request verify cached result was returned
+        return makeProfileReq(userid);
+      })
+      .then(res => {
+        assert.ok(res.headers['last-modified']);
+        assert.equal(res.headers['last-modified'], lastModified);
+        return P.delay(500);
+      })
+      .then(() => {
+        // verify cache was invalidated due to profileChangedAt update
+        mock.profileChangedAt(MOZILLA_EMAIL);
```

Comment:
Should this have `PROFILE_CHANGED_AT` as its second argument?  It looks like it might cause `profileChangedAt` to be undefined.